app/views/content_items/publication_with_tasklist_sidebar.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+        context: t("content_item.schema_name.#{@content_item.document_type}", count: 1),
+        title: @content_item.title,
+        average_title_length: "long" %>
+
+    <%= render 'shared/translations' %>
+    <% if @content_item.national_statistics? %>
+      <%= render 'shared/national_statistics_logo' %>
+    <% end %>
+
+    <%= render 'components/notice', @content_item.withdrawal_notice_component  %>
+    <%= render 'shared/metadata', content_item: @content_item %>
+    <%= render 'shared/history_notice', content_item: @content_item %>
+    <%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
+
+    <div class="grid-row">
```

Comment:
This div isn't needed and is pushing its content out of the flow of the page. Grid elements only work if they wrap grid columns, which isn't the case here.